constants/features.py

```diff
@@ -0,0 +1,12 @@
+# Local Imports
+from utils.features import Feature
+
+
+FEATURE_EARLY_ACCESS = Feature(
+    name='early_access',
+    label='Early Access',
+    description=(
+        'Allow users to request early access to the app. '
```

Comment:
Extra whitespace at the end of the string.

```suggestion
        'Allow users to request early access to the app.'
```